packages/typography/tests/Agreement.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import Agreement from '../src/Agreement';
+
+describe('Agreement', () => {
+  afterEach(cleanup);
```

Comment:
You can remove this cleanup line. The latest version of `@testing-library/react` does this for us